### DISCUSSION TITLE
posts/2020/6b9b4626/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#2] [Announcements]
# posts/2020/6b9b4626/

Visual Studio Code使用、编译、运行和调试指南。

https://blog.gztime.cc/posts/2020/6b9b4626/

**Comment by weyung** [comment]
沙发

**Comment by liaocp666** [comment]
VS Code 确实是款优秀的软件。不过我尝试了好几次，还是入坑入坑失败。我还是习惯IDEA

**Comment by ZexuanChen** [comment]
so nice guidance

**Comment by Patricia-Chi** [comment]
安装成功啦！！！！！太感谢大佬了！VSCodeCppHelper炒鸡炒鸡好用哇，卡了一上午的问题一下子就解决了(p≧w≦q)
